src/main/java/Duke.java

```diff
@@ -1,10 +1,91 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        final String DIVIDER = "    ——————————————————————————————————————————————————————————————";
+        System.out.println(DIVIDER);
+        System.out.println("    Hello! I'm Duke\n    What can I do for you?");
+        System.out.println(DIVIDER);
+        String line;
+        String[] lists = new String[100];
+        String[] flags = new String[100];
+        String[] doneTasks = new String[100];
+        String[] dates = new String[100];
```

Comment:
Based on the instructions, you need to create 1 new class in Level-3 and 3 new inherited classes from Level-4. Using the string arrays to store multiple data makes the program look more complex and decreases the quality of the code. It's better to add comments to describe the purpose of some variables, in which the names of the variables do not exactly define what it stores.